feature-libs/product-configurator/rulebased/components/attribute/types/input-field/configurator-attribute-input-field.component.ts

```diff
@@ -98,6 +98,16 @@ export class ConfiguratorAttributeInputFieldComponent
     }
   }
 
+  /**
+   * Checks if the user input has a non blank value.
+   * @returns isEmpty?
```

Comment:
Normally, we write something like this:

> returns {boolean} - 'True' if a user input is not undefined, empty or contains a blank, otherwise 'false'.